eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -160,10 +159,14 @@ object Commitments {
 
     // a node cannot spend pending incoming htlcs, and need to keep funds above the reserve required by the counterparty, after paying the fee
     // we look from remote's point of view, so if local is funder remote doesn't pay the fees
-    val fees = if (commitments1.localParams.isFunder) commitTxFee(commitments1.remoteParams.dustLimit, reduced) else Satoshi(0)
```

Comment:
I think the bug is actually here (same in `Commitments.availableForSend`. Since we consider the remote side, shouldn't we do:

```
val fees = if (commitments1.localParams.isFunder) Satoshi(0) else commitTxFee(commitments1.remoteParams.dustLimit, reduced)
```